server/ngwmn/confluence_utils.py

```diff
@@ -0,0 +1,57 @@
+
+from bs4 import BeautifulSoup
+import feedparser
+
+from . import app
+
+def pull_feed(feed_url):
+    """
+    pull page data from a my.usgs.gov confluence wiki feed
+    :param feed_url: the url of the feed, created in confluence feed builder
+    :return: the html of the page itself, stripped of header and footer
+    """
+    app.logger.debug('Parsing content from %s.', feed_url)
+    feed = feedparser.parse(feed_url)
```

Comment:
Treating this as a "service", I think it would be good to wrap this in an exception handler, log the error, and raise a ServiceException so a 503 would be returned.

Reply:
I'm thinking we might not want a 503 returned in this case especially since there are 5 feed that are fetched for a provider page. I would rather show something in the page.